msteams-platform/sbs-botbuilder-linkunfurling.yml

```diff
@@ -83,7 +83,7 @@ items:
        :::image type="content" source="~/assets/images/Teams-link-unfurling/review-plus-create.png" alt-text="Select resource Azure bot":::
        
        You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
-    
+       
```

Comment:
Not sure but it seems that the article doesn't have an edit except inserting this space. Check if this update was required. Remove, if not required.

Reply:
space removed.